swap_meet/decor.py

```diff
@@ -1,2 +1,19 @@
-class Decor:
-    pass
\ No newline at end of file
+### WAVE 5
+
+from uuid import uuid4 
+from .item import Item
+
+class Decor(Item): # Updated class to inherit from Item
+    def __init__(self, id=None, width=0, length=0, condition=0):
+        self.id = uuid4().int if id is None else id
+        self.width = 0 if width == 0 else width
+        self.length = 0 if length == 0 else length
+        super().__init__(id, condition)  # pass condition to Item
```

Comment:
See my comments in `clothing.py` about not setting the `id` here in the child class and also about reordering the code so that the parent constructor executes before setting custom child class attributes.

```suggestion
        super().__init__(id, condition)  # pass condition to Item
        self.width = 0 if width == 0 else width
        self.length = 0 if length == 0 else length
```

Reply:
Done.